### ISSUE TITLE
Just opens the image in a new tab

### ISSUE THREAD

**Issue by GeorgeWL** [sikhapol/drop-up#1]
Not sure if it works anymore ever since Atom got a native preview of image files.

Or it may just be a conflicting package

Package List:

```
-  "activate-power-mode"
-   "ask-stack"
-   "atom-bootstrap3"
-   "atom-html-preview"
-   "atom-material-syntax"
-   "atom-material-ui"
-   "autocomplete-html-entities"
-   "emmet"
-   "file-icons"
-   "fold-comments"
-   "git-clone"
-   "git-control"
-   "git-time-machine"
-   "html-entities"
-   "linter"
-   "linter-htmlhint"
-   "linter-js-standard"
-   "minimap"
-   "minimap-autohide"
-   "minimap-bookmarks"
-   "minimap-cursorline"
-   "minimap-hide"
-   "minimap-pigments"
-   "minimap-selection"
-   "package-sync"
-   "pigments"
-   "simple-drag-drop-text"
-   "time-status"
```

**Comment by yusuke024** [comment]
Hi, thank you for using my package and for reporting this.

I've installed all the packages listed above and tested it on Atom 1.72 (latest as of the time of this comment) and it still works fine.

Please note that this plugin, as of current version, will only work with markdown files or files with unknown type (as detected by Atom). With any other file types it will use Atom's default behavior and open a preview in a new tab. I'll try to come up with a configuration option to make this works with other file types in the next version. Meanwhile, if you want this to work with other file types can you tell me what type it is and how you would want it to work?

**Comment by GeorgeWL** [comment]
Well the image makes it appear as if the behaviours with any standard image filethat it will then upload to github and link.

**Comment by GeorgeWL** [comment]
Oh i see now.  Well the same behaviour on html would be good. As a quick n lazy way to upload.

**Comment by yusuke024** [comment]
I just published a new version (0.5.0) that supports .html and .css files.
Can you check it out?
Also if you have any suggestions please let me know.

Thank you again for using this and providing feedback :)
